<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_novavenda</class>
 <widget class="QDialog" name="fm_novavenda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Venda</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Arquivo_de_recursos.qrc">
    <normaloff>:/imagens/img/bolsa-de-compras.png</normaloff>:/imagens/img/bolsa-de-compras.png</iconset>
  </property>
  <widget class="QTableWidget" name="tw_listaprodutos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>811</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>671</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_excluirproduto">
      <property name="text">
       <string>Excluir Produto</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_editarproduto">
      <property name="text">
       <string>Editar Produto</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_orcamento">
      <property name="text">
       <string>Imprimir Orçamento</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_finalizarvenda">
      <property name="text">
       <string>Finalizar Venda</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>49</y>
     <width>459</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="txt_qtde">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_codproduto"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Pesquisar Produto</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Quantidade</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Código do Produto</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Valor total:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_totalvenda">
      <property name="text">
       <string>R$</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Arquivo_de_recursos.qrc"/>
 </resources>
 <connections/>
</ui>
